tests/cli/clean/test_microbial_clean.py

```diff
@@ -2,10 +2,13 @@
 import logging
 from pathlib import Path
 
+import pytest
+
 from cg.apps.tb import TrailblazerAPI
 from cg.cli.workflow.commands import clean_run_dir
 from cg.constants import Pipeline
 from click.testing import CliRunner
+from tests.conftest import microsalt_dir
```

Comment:
I dont think you need to import this? Or do you have another microsalt_dir fixture lower down in the fixture tree?